### PR TITLE
Update Editor to 2017.1.0p5 for Security Update & VS compatibility

### PR DESCRIPTION
**Pull request by StephenHodgson**
Fixes https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/849
Fixes https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/860

### CODE REVIEW

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -607,7 +611,7 @@ PlayerSettings:
       EnterpriseAuthentication: False
       HumanInterfaceDevice: False
       InputInjectionBrokered: False
-      InternetClient: False
+      InternetClient: True
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset]: Interesting that this hadn't already been set!

Comment (StephenHodgson) [reply ProjectSettings/ProjectSettings.asset]: I thought so too!

Comment (keveleigh) [reply ProjectSettings/ProjectSettings.asset]: Ohh interesting, this must be why we've never noticed it ([here](https://msdn.microsoft.com/library/windows/apps/hh464936) under **Internet and public networks**): 

> The internetClientServer capability includes the access that the internetClient capability provides, so you don't need to specify internetClient when you specify internetClientServer.

Probably doesn't hurt to keep it set though!

Comment (StephenHodgson) [reply ProjectSettings/ProjectSettings.asset]: Ahh I see.

**Comment by StephenHodgson** [comment]
I'll make a new release this weekend addressing this update.

**Pull request merged**